nsv13/code/modules/overmap/weapons/projectiles_fx.dm

```diff
@@ -563,12 +563,31 @@ Misc projectile types, effects, think of this as the special FX file.
 	tracer_type = /obj/effect/projectile/tracer/disabler
 	muzzle_type = /obj/effect/projectile/muzzle/disabler
 	impact_type = /obj/effect/projectile/impact/disabler
+	relay_projectile_type = /obj/item/projectile/beam/laser/heavylaser/phaser/relayed
+
+/obj/item/projectile/beam/laser/heavylaser/phaser/relayed
+	projectile_piercing = PASSGLASS|PASSGRILLE|PASSTABLE
+	damage = 40 //yes, the explosion will probably gib them. but if it doesn't, let's give them a chance to live, instead of smiting them with the full wraith of the Enterprise 
+
+/obj/item/projectile/beam/laser/heavylaser/phaser/relayed/on_hit(atom/target, blocked)
+	. = ..()
+	explosion(get_turf(target), 0, 2, 4, 6, ignorecap = FALSE, flame_range = 4) //I halved the BSA's explosion, and then poked it some myself - big boom
```

Comment:
ignorecap defaults to FALSE if not set in explosion args, so you can omit that for your explosion calls.